scripts/irods/test/test_iticket.py

```diff
@@ -1134,14 +1110,25 @@ def get_modification_and_creation_time():
         time.sleep(1)
         self.user.assert_icommand(['iticket', 'mod', ticket_string, 'add', 'group', 'public'])
         previous_ts = modify_ts
-        modify_ts, create_ts = get_modification_and_creation_time()
+        modify_ts, create_ts = get_modification_and_creation_time(self, ticket_string)
         self.assertGreater(modify_ts, previous_ts)
 
         # Remove ticket group restriction
         time.sleep(1)
         self.user.assert_icommand(['iticket', 'mod', ticket_string, 'remove', 'group', 'public'])
         previous_ts = modify_ts
-        modify_ts, create_ts = get_modification_and_creation_time()
+        modify_ts, create_ts = get_modification_and_creation_time(self, ticket_string)
         self.assertEqual(original_create_ts, create_ts)
         self.assertGreater(modify_ts, create_ts)
         self.assertGreater(modify_ts, previous_ts)
+
+def get_modification_and_creation_time(session, ticket_string):
```

Comment:
I think that `self` or `cls` would be a little more clear here. This isn't actually an `IrodsSession` but in fact is the instance of the test class.

Also, the indentation indicates that this is outside of the test class (i.e. top level indentation). Is that on purpose?

Reply:
I changed the parameter name to cls.

As for the indentation. It is meant to be outside the test class. This is because if it is inside the test class, it will be treated like a test case and fails.